v2/pkg/virtualbmc/ipmi_server.go

```diff
@@ -0,0 +1,124 @@
+package virtualbmc
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/cybozu-go/log"
+	"github.com/vishvananda/netlink"
+)
+
+// IPMIServer represents IPMI Server
+type IPMIServer struct {
```

Comment:
We will later add Redfish.  Please consider a more appropriate name.

Reply:
Renamed to `BMCServer`